CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml

```diff
@@ -29179,8 +29179,8 @@ First cut at header to Lewis and Short.
 <entryFree key="mercedituum" type="gloss" id="n28765"><orth lang="la" extent="full">mercēditŭum</orth>, <itype>mercenarium, quod mercede se tueatur</itype>, Paul. ex <bibl n="urn:cts:latinLit:phi1236.phi001"><author>Fest.</author> p. 124</bibl> Müll.</entryFree>
-<entryFree n="1" key="mercedonius" type="gloss" id="n28766"><orth lang="la" extent="full">mercēdŏnĭus</orth>, <itype>a, um</itype>, <pos>adj.</pos> <etym>1. merces-do</etym>, <sense level="1" n="I" id="n28766.0"><hi rend="ital">of</hi> or <hi rend="ital">belonging to the payment of wages</hi>: Mercedonios (dies) dixerunt a mercede solvendā, <hi rend="ital">on which wages are paid, pay-days</hi>, Paul. ex <bibl n="urn:cts:latinLit:phi1236.phi001"><author>Fest.</author> p. 124</bibl> Müll.—</sense><sense level="1" n="II" id="n28766.1"> <hi rend="ital">Subst.</hi>: <orth lang="la" extent="full">mercēdŏnĭus</orth>, <itype>ii</itype>, <gen>m.</gen>, <hi rend="ital">a payer of wages, paymaster</hi>: mercedonius, qui solvit mercedem; † mercedarius (<hi rend="ital">an employer</hi>), qui dat mercedem pro labore sibi impenso, <bibl><author>Gloss. Isid.</author> 37</bibl>.</sense>
-†† <sense level="3" n="2" id="n28766.2"> <orth lang="la" extent="full">Mercēdŏnĭus</orth> or <orth lang="la" extent="full">Mercēdī-nus</orth>, = <foreign lang="greek">Μερκηδόνιος, Μερκεδῖνος</foreign>, mensis, <hi rend="ital">an intercalary month of</hi> 22 <hi rend="ital">or</hi> 23 <hi rend="ital">days, inserted every two years in the calendar of Numa</hi>, Vet. Kalend. Rom. ap. Grut. 133; cf. so Laur. Lyd. de Mens. 4, 92.</sense></entryFree>
+<entryFree n="1" key="mercedonius1" type="gloss" id="n28766"><orth lang="la" extent="full">mercēdŏnĭus</orth>, <itype>a, um</itype>, <pos>adj.</pos> <etym>1. merces-do</etym>, <sense level="1" n="I" id="n28766.0"><hi rend="ital">of</hi> or <hi rend="ital">belonging to the payment of wages</hi>: Mercedonios (dies) dixerunt a mercede solvendā, <hi rend="ital">on which wages are paid, pay-days</hi>, Paul. ex <bibl n="urn:cts:latinLit:phi1236.phi001"><author>Fest.</author> p. 124</bibl> Müll.—</sense><sense level="1" n="II" id="n28766.1"> <hi rend="ital">Subst.</hi>: <orth lang="la" extent="full">mercēdŏnĭus</orth>, <itype>ii</itype>, <gen>m.</gen>, <hi rend="ital">a payer of wages, paymaster</hi>: mercedonius, qui solvit mercedem; † mercedarius (<hi rend="ital">an employer</hi>), qui dat mercedem pro labore sibi impenso, <bibl><author>Gloss. Isid.</author> 37</bibl>.</sense></entryFree>
+<entryFree n="2" key="Mercedonius2" type="greek" id="n28766"><orth lang="la" extent="full">Mercēdŏnĭus</orth> or <orth lang="la" extent="full">Mercēdī-nus</orth>, = <foreign lang="greek">Μερκηδόνιος, Μερκεδῖνος</foreign>, mensis, <hi rend="ital">an intercalary month of</hi> 22 <hi rend="ital">or</hi> 23 <hi rend="ital">days, inserted every two years in the calendar of Numa</hi>, Vet. Kalend. Rom. ap. Grut. 133; cf. so Laur. Lyd. de Mens. 4, 92.</entryFree>
```

Comment:
duplicate id number here

Reply:
Oops, thanks for catching @lcerrato . Fixed in [3980959](https://github.com/PerseusDL/lexica/pull/75/commits/398095961bddfb363b8006fbe63d8bae2ec9cc3d).